core/pom.xml

```diff
@@ -11,6 +11,11 @@
   <properties>
     <guava.version>20.0</guava.version>
     <gwt.version>2.8.0</gwt.version>
+    <junit.version>4.12</junit.version>
+    <jsr305.version>3.0.1</jsr305.version>
+    <auto-value.version>1.4.1</auto-value.version>
+    <compile-testing.version>0.10</compile-testing.version>
```

Comment:
I agree that it's fine to test with Java 8. Is that fairly straightforward to set up?